scripts/install-yum-utils.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+# install yum-utils which contains needs-restarting to check if a reboot is necessary after update
+sudo yum install -y yum-utils
+sudo yum update -y
+
+# check if it needs rebooting while yum not running
```

Comment:
Just an FYI necessary - had used this once for a cloud vm
(test ! -f /var/run/yum.pid && needs-restarting -r) || (rm -rf /var/lib/cloud/instances/;reboot)

Reply:
Im not sure why this showing as my code - some git staff with mere commit and such - but this is straight from [amazon-ecs-ami](https://github.com/kontainapp/amazon-ecs-ami/tree/muth/kontain-ami)/[scripts](https://github.com/kontainapp/amazon-ecs-ami/tree/muth/kontain-ami/scripts)/install-yum-utils.sh this file and I have not touched it, no looked at it, really. 

I think he just reboots here unconditionally. Should I add the check, you think?

Reply:
Thanks.  Sounds good.  No, let's not touch it.  Mine was because I needed it for my scenario.  Let's leave their code as is.